streaming.md

```diff
@@ -91,7 +91,8 @@ In particular, the size of the `expectedContentTypes` field (as well as the `out
 
 ### InputFrame
 `InputFrames` represent incoming data to the function. Upon reception of an `InputFrame`, an invoker
-1. SHOULD deserialize the *payload* according to the *contentType* field, which MUST be set and represent a concrete MIME type (*i.e.* not a wildcard type). The data structure that corresponds to the result of de-serialization is runtime-specific and is beyond the scope of this specification. To the best extent though, an invoker SHOULD use data structures that are *idiomatic* to the target runtime and pass them to the function if feasible. As an example, if the incoming `contentType` is `application/json` and `payload` contains the bytes of `{"foo": 5}`, then the invoker may use a *map* or a *dictionary* with key `foo`.
+1. MUST deserialize the *payload* according to the *contentType* field, which MUST be set and represent a concrete MIME type (*i.e.* not a wildcard type). The data structure that corresponds to the result of de-serialization is runtime-specific and is beyond the scope of this specification. To the best extent though, an invoker SHOULD use data structures that are *idiomatic* to the target runtime and pass them to the function if feasible. As an example, if the incoming `contentType` is `application/json` and `payload` contains the bytes of `{"foo": 5}`, then the invoker may use a *map* or a *dictionary* with key `foo`.
+If the *contentType* is not supported, an invoker MUST craft a gRPC status error of code [InvalidArgument](https://github.com/grpc/grpc/blob/2a7191abe717d3ce33b5fcbfd6bd118367499cde/doc/statuscodes.md#status-codes-and-their-use-in-grpc) and the error message MUST start with `"Invoker: Unsupported Media Type"`. An invoker MAY append error details to that message prefix.
```

Comment:
This is probably off topic for this PR, but what should the invoker do if deserialisation fails?

Reply:
idem comment above, this merits being spec'd IMO. The request/reply spec is more explicit than this, so let's align